test/mail_postfix_virtual.bats

```diff
@@ -0,0 +1,71 @@
+load 'test_helper/common'
+
+@test "checking postfix virtual alias domains no ENV set" {
+  local PRIVATE_CONFIG
+  PRIVATE_CONFIG="$(duplicate_config_for_container . )"
+  docker run -d --name mail_postfix_virtual_alias_domains \
+            -v "${PRIVATE_CONFIG}":/tmp/docker-mailserver \
+            -v "$(pwd)/test/test-files":/tmp/docker-mailserver-test:ro \
+            -h mail.my-domain.com -t "${NAME}"
+
+  teardown() { docker rm -f mail_postfix_virtual_alias_domains; }
+
+  wait_for_finished_setup_in_container mail_postfix_virtual_alias_domains
+
+  run docker exec mail_postfix_virtual_alias_domains postconf virtual_alias_domains
+  assert_output "virtual_alias_domains = ???"
```

Comment:
All this file is not tested because the tests do not run on my machine..
I will have to debug with travis ..
If anyone can help :pray: